instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
@bradenmacdonald The problem with the current approach is that Django wants to create a new migreation whenever `DEFAULT_ADMIN_ORGANIZATION` is changed, which is the reason why we avoid setting default values based on settings in the field constructor.  We mostly got away with this because we have all set this to the empty string in our dev environments, but on production it was always complaining about pending changes.